gamestonk_terminal/cryptocurrency/overview/overview_controller.py

```diff
@@ -1772,9 +1802,16 @@ def menu():
     while True:
         # Get input command from user
         if session and gtff.USE_PROMPT_TOOLKIT:
-            completer = NestedCompleter.from_nested_dict(
-                {c: None for c in overview_controller.CHOICES}
-            )
+            choices: dict = {c: {} for c in overview_controller.CHOICES}
+
+            choices["wfpe"]["-c"] = {
```

Comment:
@piiq @jmaslek we need to step up our UX overall on the terminal using this and difflib!

We need to go through our controllers 1 by 1 and tackle this